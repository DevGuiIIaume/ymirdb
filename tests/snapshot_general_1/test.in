set a 1
set b a 2
set c 3
set d c 4
set e 5
push e a c
list entries
snapshot
checkout 1
forward a
forward b
forward c
forward d
forward e
backward a
backward b
backward c
backward d
backward e
bye
