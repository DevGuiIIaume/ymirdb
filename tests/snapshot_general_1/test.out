> ok

> ok

> ok

> ok

> ok

> ok

> e [c a 5]
d [c 4]
c [3]
b [a 2]
a [1]

> saved as snapshot 1

> ok

> nil

> a

> nil

> c

> a, c

> b, e

> nil

> d, e

> nil

> nil

> bye
